Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.Scale.cs

```diff
@@ -127,7 +127,14 @@ public static partial class AnimationExtensions
                 var animation = compositor.CreateVector3KeyFrameAnimation();
                 animation.Duration = TimeSpan.FromMilliseconds(duration);
                 animation.DelayTime = TimeSpan.FromMilliseconds(delay);
-                animation.InsertKeyFrame(1f, scaleVector);
+                if (easingType == EasingType.Default)
```

Comment:
This if block seems to be duplicated everywhere (with some variance). Is it possible to extract this into a method that does this?

Reply:
InsertKeyFrame on CompositionAnimation has several overloads that are used so we would need to create several methods (which would complicate things more)